src/jit/lower.cpp

```diff
@@ -1546,7 +1546,7 @@ void Lowering::LowerCall(GenTree* node)
             }
 #endif
 #if defined(_TARGET_AMD64_) && defined(FEATURE_READYTORUN_COMPILER)
-            if (call->gtEntryPoint.addr != nullptr)
+            if ((call->gtEntryPoint.addr != nullptr) && !call->IsFastTailCall())
```

Comment:
It turns out that this ifdef can be safely removed now. It was a workaround for a problem that has been fixed since then.